src/clients/kdump.rb

```diff
@@ -735,20 +735,20 @@ def cmdKdumpShow(options)
 
 
     # Only numbers are allowed as allow_mem_high and allow_mem_low values
-    ALLOC_MEM_REGEXP = /\D/
+    ALLOC_MEM_REGEXP = /\A\d+\z/
 
     def cmdKdumpStartup(options)
       options = deep_copy(options)
       if Ops.get(options, "enable") != nil &&
           Ops.get(options, "alloc_mem") != nil
-        Kdump.add_crashkernel_param = true
         alloc_mem_low, alloc_mem_high = options["alloc_mem"].split(',')
-        Kdump.allocated_memory = { low: alloc_mem_low, high: alloc_mem_high }
         unless alloc_mem_low =~ ALLOC_MEM_REGEXP &&
                 (alloc_mem_high.nil? || alloc_mem_high =~ ALLOC_MEM_REGEXP)
           CommandLine.Error(_("Invalid allocation memory parameter"))
```

Comment:
"allocation memory" is wrong, it should be "memory allocation"; but even better, quote the actual parameter name